Search-Engine-Bandits/nutshell
# Jl nutshell setup

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change
1.  Added button to call form for new event
2.  Click event on button to create form on dom
3.  Click event on "submit event" button to post data to api and then populate dom with all events
4.  fetch call to post events to dom is done in ascending order so next event upcoming will be at the top
5.  posted data from json file to json.example file for team testing 

Please delete options that are not relevant.

- [x ] New feature (non-breaking change which adds functionality)
- [x ] This change requires a documentation update

# Testing Instructions

to test, git fetch --all
branch is jl-nutshell-setup
open database.json.example & copy data from "events" section
paste events data into your database.json in the events arrray
run npm start & you should see existing events populating on dom, and be able to create a new event (delete & edit buttons not functional yet)



# Checklist:

- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
